pydatastructs/trees/btree.py

```diff
@@ -0,0 +1,214 @@
+import bisect
+from pydatastructs.utils import BNode
+
+__all__ = [
+    'BTree'
+]
+
+class BTree(object):
+    """
+    Represents B-tree.
+
+    Parameters
+    ==========
+
+    order : int
+        Optional, by default 1.
+        Order of initial TreeNode in B-Tree.
+
+    Examples
+    ========
+
+    >>> from pydatastructs.trees.btree import BTree
+    >>> bt = BTree(5)
+    >>> bt.insert(1)
+    >>> bt.insert(2)
+    >>> bt.insert(3)
+    >>> bt.insert(4)
+    >>> list(bt)
+    [1, 2, 3, 4]
```

Comment:
BTree is a self balancing tree so input should be arranged accordingly and printing Tree should give be of form of any order(Post/Pre/In)
(Read more here)[https://www.geeksforgeeks.org/introduction-of-b-tree-2/]

Reply:
I'm not getting your point it's just example for users to see

Reply:
For you reference it's already in order traversal as per the reference https://www.geeksforgeeks.org/introduction-of-b-tree-2/